modules/isolation_segment/security_groups.tfwip

```diff
@@ -0,0 +1,906 @@
+# Isolation Segment Firewall Rules: https://docs.pivotal.io/pivotalcf/2-3/adminguide/routing-is.html#config-firewall
```

Comment:
Yes. I think this could be terraformed with a little more expertise. I also did this in a somewhat ad-hoc way as per the documentation requirements while also assuming that those requirements came from experts on iso segments.

Reply:
We recently merged a PR for gcp that has similar bits and is very well documented. If you could take a look and make the necessary changes would be of incredible value and help for us.
That the PR: https://github.com/pivotal-cf/terraforming-gcp/pull/113

Reply:
@malston Have you had time to take a look at these?

Reply:
Not yet @cdutra . I will look at them tomorrow -- 12/7